ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,59 @@
+<!--
+Compensation requests have a specific format that enables them to be parsed programmatically for more efficient budgeting. Please make sure you stick to the template. A linter will evaluate your request after you remove [WIP] from the issue title and notify you of any issues.
+
+See the following link for details:
+https://bisq.wiki/Making_a_compensation_request
+
+Feel free to browse other requests if you're not sure about something, or reach out in #compensation on Keybase (https://keybase.io/team/bisq).
+-->
+
+
 ## Summary
 
-> _Specify the total amount of BSQ you are requesting:_
+> _Specify the total amount of BSQ you are requesting, along with the USD total and BSQ/USD rate:_
 
- - BSQ requested:
- 
- <!--
- Don't forget to make your compensation request in the actual Bisq DAO! See the following link for details:
- https://docs.bisq.network/compensation.html
- -->
+- **BSQ requested**: **`[BSQ amount]`** <!-- [BSQ amount] should be: USD amount / BSQ rate -->
+- USD requested: [USD amount] <!-- [USD amount] should be: total of all contributions listed in "contributions delivered" table below -->
+- BSQ rate: [rate] USD per BSQ <!-- [rate] should be the rate used to determine the BSQ amount above, as specified in the cycle's rate announcement (https://bisq.wiki/Compensation_Maintainer#Announce_BSQ-USD_rate) -->
 
 
 ## Contributions delivered
 
-> _Provide links to contributions you have [delivered](https://github.com/bisq-network/proposals/issues/19), the amount of BSQ you are requesting for each, and any comments that will help stakeholders understand its value._
+> _Add contributions you have [delivered](https://github.com/bisq-network/proposals/issues/19) here as new rows in the table below. See comments below the table for guidance._
 
+| Title | Team | USD | Link | Notes |
+| --- | --- | --- | --- | --- |
 
-## Contributions in progress
+<!-- 
+
+Each contribution should be a new line-item in the table above. 
+
+Title - optional, string.
+Team - optional, must one of the following: admin, dev, growth, ops, support (case-insensitive). If the line-item is for a role, please add a `*`. See Example 2 below.
+USD - optional, number. Please do not specify 'USD', as the column title makes units clear.
+Link - optional, string.
+Notes - optional, string.
 
-> _Provide links to work you're involved with that is still [in progress](https://github.com/bisq-network/proposals/issues/19). **This section is optional,** and is for your own benefit in keeping track of what you're doing and keeping other contributors up to date with the same._
+Example 1 - typical contribution:
 
+| Add local Bitcoin node configuration detection | dev | 240 | https://github.com/bisq-network/bisq/pull/3982
 
-## Roles performed
+Example 2 - role (note the asterisk):
```

Comment:
What is the significance of the asterisk?

Reply:
No significance to the bot, it ignores any asterisk after the team name.

Reply:
Yeah the bot currently ignores it, but its purpose is to mark line-items for _roles_ (so that they can be distinguished from regular _work-delivered_ line-items). 

I figure this will be useful when we start aggregating qualitative assessments of issuance. Role contributions can be thought of as the project's 'overhead', and evaluating tracking its proportion of overall issuance could be useful.

Reply:
Ok, I think that without further change, people won't get that they should mark role entries with an asterisk. It's just too subtle / not explicit enough here. But I wouldn't personally hold up the merge on it at this point. If it's something that people _should_ do, I'd recommend saying that clearly.

Reply:
Addressed with https://github.com/bisq-network/compensation/pull/585/commits/1b94f34488a592662cba987a92d39b77c7c6a58a. Will probably still need to remind people, lead by example, etc., but role owners are mostly core/active contributors so it should be ok.